polylogue/cli/render.py

```
@@ -78,20 +78,22 @@ def _truthy(val: str) -> bool:
         meta=meta,
     )
-    if getattr(args, "max_disk", None):
+    disk_estimate = bool(getattr(args, "disk_estimate", False))
+    max_disk = getattr(args, "max_disk", None)
+    if disk_estimate or max_disk:
         projected = len(inputs) * 5 * 1024 * 1024  # rough 5MiB per file heuristic
-        from ..util import preflight_disk_requirement
-
+        try:
+            free_bytes = shutil.disk_usage(Path.cwd()).free
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)